app/controllers/student_requests_controller.rb

```diff
@@ -27,13 +27,20 @@ def cancel
   end
 
   def create
-    @student_request = StudentRequest.create!(student_request_params)
+    @student_request = StudentRequest.new(student_request_params)
+    @student_request.state = "active"
+    @student_request.save!
     flash[:notice] = "Student Request was successfully created."
     redirect_to student_requests_path
   end
-
+  
   def edit
-    @student_request = StudentRequest.find params[:request_id]
+    @student_request = StudentRequest.find params[:id]
+    @student_request.state = "false"
```

Comment:
use better name for showing state in case of deletion of request? either deleted or withdrawn or cancelled or inactive? I will suggest withdrawn as it will be from perspective of student. Also, don't use them as normal strings. Instead use the constant variables defined in student_request model class ...